modules/tinymce/src/plugins/paste/test/ts/atomic/NewlinesTest.ts

```diff
@@ -0,0 +1,60 @@
+import { describe, it } from '@ephox/bedrock-client';
```

Comment:
This test was an `atomic` test but it was incorrectly placed in the `browser` directory.